### PR TITLE
Flyway DB migrations

### PR DESCRIPTION
**Pull request by mishraomp**
Add flyway init-container pattern

---

Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend](https://quickstart-openshift-1099-backend.apps.silver.devops.gov.bc.ca/) available
[Frontend](https://quickstart-openshift-1099-frontend.apps.silver.devops.gov.bc.ca/) available

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/quickstart-openshift/actions/workflows/merge-main.yml)

### CODE REVIEW

**Comment by paulushcgcj** [comment]
@mishraomp can I suggest a more generic approach? On [old growth](https://github.com/bcgov/nr-old-growth) I used a similar approach, but instead of baking the migration inside flyway, I just added the migrations as part of a [config map](https://github.com/bcgov/nr-old-growth/blob/main/.github/openshift/deploy.backend.yml#LL66C3-L86C119). This way the [init container](https://github.com/bcgov/nr-old-growth/blob/main/.github/openshift/deploy.backend.yml#L142) is just a "template" and the migration will be tied to the application itself, making it easier to be copied/replicated by other projects.

**Comment by mishraomp** [comment]
I believe , multiple imagestream objects in a single deployment yaml file is causing the deploy to break. thoughts @DerekRoberts ?

**Comment by DerekRoberts** [comment]
@mishraomp I haven't even narrowed it down that far yet.  Multiple image streams sure should be fine.

**Comment by DerekRoberts** [comment]
@mishraomp Nope, you were right!  PR for bug merged, here for reference.  https://github.com/bcgov-nr/action-deployer-openshift/pull/26

File: backend/db/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM flyway/flyway:9.10-alpine
```
[the remark this answers is not in the document]
Comment (DerekRoberts) [reply backend/db/Dockerfile:1]: This isn't used by OpenShift/Kubernetes.  Completely optional for us.

**Comment by mishraomp** [comment]
@DerekRoberts are we good to move, can you approve?

**Review by DerekRoberts — APPROVED** [review]
Good pattern building and feature addition!  :)

**Pull request merged**